### ISSUE TITLE
COSMIC coding mutations VCF no longer available (v94 deprecated)

### ISSUE THREAD

**Issue by ivadym** [Clinical-Genomics/BALSAMIC#1145]
**Description**

v94 version of the coding file is no longer available:
https://github.com/Clinical-Genomics/BALSAMIC/blob/b9c59a4a0beacc11a75a502ef46a7db353aa982b/BALSAMIC/constants/reference.py#L72-L79

**Suggested solution**

Update to one of the available versions:
```
curl https://cancer.sanger.ac.uk/cosmic/file_download/GRCh37/cosmic
[
  "GRCh37/cosmic/v95",
  "GRCh37/cosmic/v96",
  "GRCh37/cosmic/v97",
  "latest"
]
```
```
curl https://cancer.sanger.ac.uk/cosmic/file_download/GRCh38/cosmic
[
  "GRCh38/cosmic/v95",
  "GRCh38/cosmic/v96",
  "GRCh38/cosmic/v97",
  "latest"
]
```

**This can be closed when:**

- [ ] Version updated and the VCF file is downloaded

**Comment by pbiology** [comment]
This bug will be fixed by #1148

**Comment by ivadym** [comment]
Completed https://github.com/Clinical-Genomics/BALSAMIC/pull/1147
